configextractor.py

```diff
@@ -10,6 +11,54 @@
 cl_engine = forge.get_classification()
 
 HEURISTICS_MAP = {"malware": 1, "safe": 2}
+# This dict contains fields that we care about, and the corresponding tag if they exist
+FIELD_TAG_MAP = {
+    # TODO: sometimes a domain is returned for address, which should be tagged as network.dynamic.domain
+    # TODO: sometimes an IP:port is returned for address, which should be split into IP and tagged as network.dynamic.ip and port and tagged as network.port
+    'address': 'network.dynamic.uri',
+    'c2_address': 'network.dynamic.uri',
+    'c2_socketaddress': None,
+    'c2_url': 'network.dynamic.uri',
+    'credential': None,
+    'directory': None,
+    'email_address': None,
+    'event': None,
+    'filename': None,
+    'filepath': 'file.path',
+    'ftp': None,
+    'guid': None,
+    'injectionprocess': None,
+    'interval': None,
+    'key': None,
+    'listenport': None,
+    'missionid': None,
+    'mutex': 'dynamic.mutex',
+    'outputfile': None,
+    'password': 'file.string.extracted',
+    'pipe': None,
+    # "port": "network.port",  # TODO: What is returned is a list that contains non-integer types, which must be avoided when tagged for network.pprt
```

Comment:
this is expected, mwcp framework defines port as a list of port numbers and protocols. https://github.com/Defense-Cyber-Crime-Center/DC3-MWCP/blob/0d27dea113751ef1aff727aaa2cf13c709e6beab/mwcp/config/fields.json#L240

Reply:
Oh I know, but we still need to find a way to parse this port list accordingly such that we can tag the port numbers with `network.port` and the protocols used with `network.protocol`